ecl/eclagent/eclagent.cpp

```diff
@@ -2148,6 +2110,11 @@ void EclAgent::doProcess()
         e->Release();
     }
     DBGLOG("Workunit written complete");
+
+    if (getClusterType(clusterType)==ThorLCRCluster)
+    {
+        runWorkunitAnalyser(*wuRead, getComponentConfigSP(), nullptr, true, calculateThorCost(3600000, getNodes()));
```

Comment:
You could create a
`calculateThorCostPerHour(numWorkers)
`
 function to clarify the code (since it isn't immediately obvious, and is repeated elsewhere).